lib/codecs/src/encoding/mod.rs

```diff
@@ -79,7 +93,7 @@ impl From<NewlineDelimitedEncoderConfig> for FramingConfig {
 
 impl FramingConfig {
     /// Build the `Framer` from this configuration.
-    pub const fn build(self) -> Framer {
+    pub fn build(self) -> Framer {
```

Comment:
Just curious what changed here to make this no longer `const`.

Reply:
`tokio_util::codec::LengthDelimitedCodec` is not `const` and therefore by extentions all the methods that use it can not be `const` any more. I only removed it from places where it would not compile, otherwise clippy would have complained here